lib/ClangImporter/SortedCFDatabase.def.gyb

```diff
@@ -27,7 +27,7 @@ lineForName = {}
 
 # Load the data file.
 with open(CFDatabaseFile, 'rb') as f:
-  for line in f:
+  for line in [line_bytes.decode('utf-8') for line_bytes in f]:
```

Comment:
You should make this:

```python
for line in (line_bytes.decode('utf-8') for line_bytes in f):
```

so that it utilizes generators instead of building up a list unnecessarily

https://www.python.org/dev/peps/pep-0289/